sites/public/pages/index.tsx

```diff
@@ -1,99 +1,92 @@
-        <div className="flex absolute w-full flex-col items-center">
-          <SiteAlert type="alert" className={alertClasses} />
-          <SiteAlert type="success" className={alertClasses} timeout={30000} />
-        </div>
-        {this.state.alertMessage && (
-          <AlertBox className="" onClose={() => this.closeAlert()} type={this.state.alertType}>
-            {this.state.alertMessage}
-          </AlertBox>
-        )}
-        <Hero
-          title={heroTitle}
-          buttonTitle={t("welcome.seeRentalListings")}
-          buttonLink="/listings"
-          listings={this.props.listings}
-        />
-        <div className="homepage-extra">
-          <MarkdownSection fullwidth={true}>
-            <>
-              <p>{t("welcome.seeMoreOpportunities")}</p>
-              <LinkButton href="/additional-resources">
-                {t("welcome.viewAdditionalHousing")}
-              </LinkButton>
-            </>
-          </MarkdownSection>
-        </div>
-        <ConfirmationModal setSiteAlertMessage={this.setSiteAlertMessage} />
-      </Layout>
-    )
+  const metaDescription = t("pageDescription.welcome", { regionName: t("region.name") })
+  const metaImage = "" // TODO: replace with hero image
+  const alertClasses = "flex-grow mt-6 max-w-6xl w-full"
+  return (
+    <Layout>
+      <Head>
+        <title>{t("nav.siteTitle")}</title>
+      </Head>
+      <MetaTags title={t("nav.siteTitle")} image={metaImage} description={metaDescription} />
+      <div className="flex absolute w-full flex-col items-center">
+        <SiteAlert type="alert" className={alertClasses} />
+        <SiteAlert type="success" className={alertClasses} timeout={30000} />
+      </div>
+      {alertInfo.alertMessage && (
+        <AlertBox
+          className=""
+          onClose={() => setAlertInfo(blankAlertInfo())}
+          type={alertInfo.alertType}
+        >
+          {alertInfo.alertMessage}
+        </AlertBox>
+      )}
+      <Hero
+        title={heroTitle}
+        buttonTitle={t("welcome.seeRentalListings")}
+        buttonLink="/listings"
+        listings={props.listings}
+      />
+      <div className="homepage-extra">
+        <MarkdownSection fullwidth={true}>
+          <>
+            <p>{t("welcome.seeMoreOpportunities")}</p>
+            <LinkButton href="/additional-resources">
+              {t("welcome.viewAdditionalHousing")}
+            </LinkButton>
+          </>
+        </MarkdownSection>
+      </div>
+      <ConfirmationModal
+        setSiteAlertMessage={(alertMessage, alertType) => setAlertInfo({ alertMessage, alertType })}
+      />
+    </Layout>
+  )
+}
+
+export async function getStaticProps() {
+  let listings = []
+  try {
+    const response = await axios.get(process.env.listingServiceUrl)
+    listings = response.data
+  } catch (error) {
+    console.error(error)
   }
+
+  return { props: { listings } }
 }
```

Comment:
This is pretty cool. For anyone else wondering what the purpose of this was: https://nextjs.org/learn/basics/data-fetching/implement-getstaticprops